### DISCUSSION TITLE
stylining a report - "fancy header" - line in header and footer

### DISCUSSION THREAD

**Discussion by rainer-rq-koelle** [typst/typst#3464] [Q&A]
I am looking into using Typst for producing reports at work (probably in combination with Quarto) - chapeau for the great work. I am wondering how one would define a "fancy header", i.e. inserting a header and footer separation line. 
My gut feeling is that one could place a box in the header and only color its bottom ... footer vice-versa. But I am unsure on how to get this defined in typst. 
Thanks for pointing me in the right direction.
Servus from Brussels!

**Comment by istudyatuni** [comment]
From [discord support thread](https://discord.com/channels/1054443721975922748/1209522392058044436):

```typ
#set page(header: "header text" + line(length: 100%))
```
